org.mixedrealitytoolkit.spatialmanipulation/BoundsControl/BoundsControl.cs

```diff
@@ -683,7 +683,7 @@ private bool ComputeBounds(bool isSecondPass = false)
         /// Called by <see cref="BoundsHandleInteractable"/> from its OnSelectExited.
         /// Routes the XRI event data through, as well as a reference to itself, the selected handle.
         /// </summary>
-        internal void OnHandleSelectExited(BoundsHandleInteractable handle, SelectExitEventArgs args)
```

Comment:
protected virtual internal? BoundsHandleInteractable needs to be able to access this